src/codegen.cpp

```diff
@@ -4723,7 +4719,7 @@ static std::unique_ptr<Module> emit_function(jl_lambda_info_t *lam, jl_llvm_func
                                              0,
                                              true,
                                              nullptr);
-                builder.SetCurrentDebugLocation(DebugLoc::get(0, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
+                builder.SetCurrentDebugLocation(DebugLoc::get(inlfline, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
             }
```

Comment:
I don't think we have any intermediate instructions to hide, so it's fine to fold those two calls (the DebugLoc set in the builder is only side-effecting on the instructions inserted by that builder). Just noting that if we want to do work to create a call frame or such, those instructions should report a line of 0 to the debugger.